doc/qos/dynamically-headroom-calculation.md

```diff
@@ -787,7 +801,6 @@ The values used in the above procedure are fetched from the following table:
 - `port mtu`: PORT|\<port name\>|mtu, default value is `9100`
 - `gearbox delay`: PERIPHERIAL_TABLE|\<gearbox name\>|gearbox_delay
 - `mac/phy delay`: ASIC_TABLE|\<asic name\>|mac_phy_delay
-- `peer response`: ASIC_TABLE|\<asic name\>|peer_response_time

```

Comment:
This still needs to be there. You use this value if the speed is something other than the one defined in the table above

Reply:
@neethajohn, rephrased, please have a look